confirm/validator.py

```diff
@@ -2,131 +2,146 @@
 Main module for the validation functionalities.
 """
 from confirm.utils import get_most_probable_typo
+from confirm.utils import load_config_file, load_schema_file
 
 
 VALID_TYPES = ('int', 'float', 'bool', 'list', 'str')
 
 
-def validate_config(config, schema, error_on_deprecated=False):
+def load_validation_from_files(schema_file_path, config_file_path):
+    schema = load_schema_file(open(schema_file_path, 'r'))
```

Comment:
We don't do any more error handling than the Yaml library, no.
What would you suggest?